### PR TITLE
Fix: cutting long file names to avoid errors when saving

### PR DESCRIPTION
**Pull request by Napuh**
# Description

I've added a limit into the mp4 file name to avoid errors when saving files with very long filenames due to a long reddit post title.

# Issue Fixes

This does not fix any issue I've seen in the repo but fixes a problem I've encountered when testing the code.

None

# Checklist:

- [ x] I am pushing changes to the **develop** branch
- [ x] I am using the recommended development environment
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have formatted and linted my code using python-black and pylint
- [ x] I have cleaned up unnecessary files
- [ x] My changes generate no new warnings
- [ x] My changes follow the existing code-style
- [ x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

In this particular post (https://www.reddit.com/r/explainlikeimfive/comments/vxwol7/eli5_in_light_of_everyone_losing_their_minds_over/) the filename generated is too long and throws an error when saving. 
This modification to the name_normalize() method solves this.

### CODE REVIEW

**Comment by CordlessCoder** [comment]
One question: but why 70?

**Comment by Napuh** [comment]
Well, that´s an arbitrary number ngl. I figured out that 70 its enough to distinguish files. 
If you think is better to change it to another number let me know.

**Review by JasonLovesDoggo — APPROVED** [review]

**Comment by NeoV55** [comment]
```
[04:55:17]  Video Will Be: 53 Seconds Long                                                            final_video.py:83
Moviepy - Building video assets/temp/temp.mp4.
MoviePy - Writing audio in tempTEMP_MPY_wvf_snd.mp4
MoviePy - Done.
Moviepy - Writing video assets/temp/temp.mp4

Moviepy - Done !
Moviepy - video ready assets/temp/temp.mp4
Moviepy - Running:
>>> "+ " ".join(cmd)
Moviepy - Command returned an error
Traceback (most recent call last):
  File "C:\Users\User\RedditVideoMakerBot\main.py", line 65, in <module>
    run_many(config["settings"]["times_to_run"])
  File "C:\Users\User\RedditVideoMakerBot\main.py", line 56, in run_many
    main()
  File "C:\Users\User\RedditVideoMakerBot\main.py", line 48, in main
    make_final_video(number_of_comments, length, reddit_object, bg_config)
  File "C:\Users\User\RedditVideoMakerBot\video_creation\final_video.py", line 144, in make_final_video
    ffmpeg_extract_subclip(
  File "C:\Users\User\anaconda3\lib\site-packages\moviepy\video\io\ffmpeg_tools.py", line 64, in ffmpeg_extract_subclip
    subprocess_call(cmd)
  File "C:\Users\User\anaconda3\lib\site-packages\moviepy\tools.py", line 54, in subprocess_call
    raise IOError(err.decode('utf8'))
OSError: ffmpeg version 4.2.2 Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 9.2.1 (GCC) 20200122
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-libmfx --enable-amf --enable-ffnvcodec --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth --enable-libopenmpt
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'assets/temp/temp.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf58.29.100
  Duration: 00:00:54.13, start: 0.000000, bitrate: 2413 kb/s
    Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1080x1920, 2216 kb/s, 30 fps, 30 tbr, 15360 tbn, 60 tbc (default)
    Metadata:
      handler_name    : VideoHandler
    Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 195 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
results/todayilearned/TIL upon request Martin Luther King Jr gave his copy of the I have a dream speech he had just delivered to the then security guard George Raveling Raveling has been offered as much as 3 million for the original copy but he has refused all offers.mp4: No such file or directory
```

hi guys , still facing the same issue ... tried almost every workaround/fix but with no success ... any help is very much appreciated, thank you guys.

**Comment by Darkatek7** [comment]
CHange it to 200 Please, doesn't work with 200
